glean-core/ios/Glean/Glean.swift

```diff
@@ -71,6 +75,15 @@ public class Glean {
             return glean_initialize(&cfg)
         }
 
+        // If any pings were registered before initializing, do so now
+        for ping in self.pingTypeQueue {
+            self.registerPingType(ping)
+        }
+
+        if !Dispatchers.shared.testingMode {
+            self.pingTypeQueue.removeAll()
+        }
```

Comment:
Do we need any sort of synchronization here?  So we don't add and flush ping types at the same time...  (Not saying we do, just want to confirm...)

Reply:
We'll verify in a follow-up